src/components/EventCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { ReactElement } from "react";
+import ".././styles.css";
+import PinSVG from ".././public/Pin.svg";
+import TextIconSVG from ".././public/TextIcon.svg";
+import MeatballMenuSVG from ".././public/MeatballMenu.svg";
+
+export interface EventCardProps {
+  startTime: Date;
+  endTime?: Date;
+  name: string;
+  location: string;
+  description: string;
+}
+
+/**
+ * Renders a single event in the feed
+ */
+const EventCard = ({
+  startTime,
+  endTime,
+  name,
+  location,
+  description,
+}: EventCardProps): ReactElement => {
+  let startTimeString: string = startTime.toLocaleString("en-US", {
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+  let endTimeString: string | undefined = endTime
+    ? endTime.toLocaleString("en-US", {
+        hour: "numeric",
+        minute: "numeric",
+        hour12: true,
+      })
+    : undefined;
+
+  return (
+    <>
+      <div className="pl-4 pt-4 pr-6 w-4/5">
+        <span className="font-sans">
+          {startTimeString + (endTime ? " to " + endTimeString : "")}
+        </span>
+        <div className="flex flex-row justify-between items-start mb-4">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+            {name}
+          </div>
+          <img
+            src={MeatballMenuSVG}
+            alt="Menu svg"
+            aria-label="Open Event Card details"
+          />
+        </div>
+        <div className="flex flex-row items-start mb-4">
+          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+          <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
```

Comment:
Instead of `break-words`, would we maybe want to use `break-normal` for readability? If words are allowed to break mid-word, it may be harder to read.

Reply:
The impression I got from reading tailwind docs is that `break-normal` will preserve words no matter what and one really long word could spill off the page, while `break-words` will only break a word mid-word if it's a particularly long word that's longer than the line length.

So it's probably very unlikely for `break-words` to have to break a word, and in the case that it does happen I think would make sense to use `break-words`. Lmk if you have any thoughts on this and if it still makes sense to use `break-normal`!

Reply:
Ahh okay that makes sense! I think this works then!